modules/twinkleunlink.js

```diff
@@ -164,7 +164,7 @@ Twinkle.unlink.callbacks = {
 					apiobj.params.form.append( { type:'header', label: 'File usage' } );
 					namespaces = [];
 					$.each(Twinkle.getPref('unlinkNamespaces'), function(k, v) {
-						namespaces.push(Morebits.wikipedia.namespacesFriendly[v]);
+						namespaces.push(v === 0 ? '(Article)' : mw.config.get('wgFormattedNamespaces')[v]);
```

Comment:
```suggestion
						namespaces.push(v === '0' ? '(Article)' : mw.config.get('wgFormattedNamespaces')[v]);
```
Sorry, led you slightly astray; `$.each` casts v as a string, while the raw `0` gets treated as `number`